src/main/java/com/team/buddyya/feed/dto/response/FeedCreateResponse.java

```diff
@@ -0,0 +1,10 @@
+package com.team.buddyya.feed.dto.response;
+
+public record FeedCreateResponse(
+        Long id
+) {
```

Comment:
P3: 컨벤셩 상 record의 필드 하나인 경우 
```suggestion
public record FeedCreateResponse(Long id) {
}
```
이렇게 통일하기로 했어요!

Reply:
네 알겠습니다!